airbyte-integrations/connectors/source-pinterest/acceptance-test-config.yml

```diff
@@ -62,4 +62,4 @@ acceptance_tests:
           disable_for_version: 0.1.2
         spec_path: source_pinterest/spec.json
 connector_image: airbyte/source-pinterest:dev
-test_strictness_level: high
+test_strictness_level: low
```

Comment:
`strictness_level: high` -  required for GA connectors

Reply:
Sorry I lower the level to test only test_read. Rolling back right now.